tests/kernel/boot_page_table/src/boot_page_table.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <mmustructs.h>
+#include <ztest.h>
+#include "boot_page_table.h"
+
+MMU_BOOT_REGION(START_ADDR_RANGE, ADDR_SIZE, REGION_PERM);
```

Comment:
add multiple regions into the test. as of now it is testing a single memory region.
Create multiple memory regions and validate them with the below APIs.

Reply:
multiple memory regions has been created and added to validate upon.